setup.py

```diff
@@ -1,459 +1,22 @@
-
-setup (name = 'phoebe',
-       version = '2.4.11',
-       description = 'PHOEBE 2.4',
-       long_description=long_description,
-       author = 'PHOEBE development team',
-       author_email = 'phoebe-devel@lists.sourceforge.net',
-       url = 'http://github.com/phoebe-project/phoebe2',
-       classifiers=[
-            'Development Status :: 5 - Production/Stable',
-            'Intended Audience :: Science/Research',
-            'Topic :: Software Development :: Build Tools',
-            "Natural Language :: English",
-            "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
-            "Topic :: Scientific/Engineering",
-            "Topic :: Software Development :: Libraries :: Python Modules",
-            'Programming Language :: Python :: 3',
-            'Programming Language :: Python :: 3.6',
-            'Programming Language :: Python :: 3.7',
-            'Programming Language :: Python :: 3.8',
-            'Programming Language :: Python :: 3 :: Only',
-        ],
-       python_requires='>=3.6, <4',
-       download_url = 'https://github.com/phoebe-project/phoebe2/tarball/2.4.2',
-       packages = ['phoebe',
-                   'phoebe.parameters', 'phoebe.parameters.solver', 'phoebe.parameters.figure',
-                   'phoebe.frontend',
-                   'phoebe.constraints',
-                   'phoebe.dynamics',
-                   'phoebe.distortions',
-                   'phoebe.algorithms',
-                   'phoebe.atmospheres',
-                   'phoebe.backend',
-                   'phoebe.solverbackends', 'phoebe.solverbackends.ebai', 'phoebe.solverbackends.knn',
-                   'phoebe.utils',
-                   'phoebe.helpers',
-                   'phoebe.pool',
-                   'phoebe.dependencies', 'phoebe.dependencies.autofig', 'phoebe.dependencies.nparray', 'phoebe.dependencies.distl', 'phoebe.dependencies.crimpl', 'phoebe.dependencies.unitsiau2015',
-                   'phoebe.dependencies.ligeor',
-                   'phoebe.dependencies.ligeor.ebai', 'phoebe.dependencies.ligeor.ebai.database', 'phoebe.dependencies.ligeor.eclipse', 'phoebe.dependencies.ligeor.models', 'phoebe.dependencies.ligeor.utils'],
-       install_requires=['numpy>=1.15','scipy>=1.4','astropy>=3.0', 'corner', 'pytest', 'requests', 'tqdm', 'python-socketio[client]']+['flask', 'flask-cors', 'flask-socketio==4.3.*', 'gevent-websocket'],
-       package_data={'phoebe.atmospheres':['tables/wd/*', 'tables/passbands/*'],
-                     'phoebe.frontend':['default_bundles/*.bundle'],
-                     'phoebe.solverbackends.ebai': ['*.data', '*.weights'],
-                     'phoebe.solverbackends.knn': ['*.knn']
-                    },
-       ext_modules = ext_modules,
-       scripts=None if  _env_variable_bool('PHOEBE_SKIP_SCRIPTS', False) else ['client-server/phoebe-server', 'client-server/phoebe-autofig'],
```

Comment:
I missed this - but it [seems this removed support for installing these scripts](https://github.com/phoebe-project/phoebe2-ui/issues/26)... can this be reintroduced in the current setup?